### PR TITLE
Add vue recomended linter rules

### LINKED ISSUE #236: Different timestamp types
Comment and tip have different type of timestamp, I suggest to choose one of them.
- tip timestamp type is number: 1584484988877
- comment timestamp type is a String: 2020-03-17 13:09:14.485 +00:00

### LINKED ISSUE #238: Different types of amount
In AeAmount and FiatValue components amount in sended in a different types. String or Object

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #236 
fixes #238

### CODE REVIEW

File: src/components/layout/RightSection.vue [hunk 1]
```diff
@@ -1,44 +1,69 @@
...
             <div>
               <span class="value">
-                <ae-amount :amount="data.amount" :round="2"></ae-amount>
+                <ae-amount
+                  :amount="data.amount"
+                  :round="2"
+                />
               </span>
-              <fiat-value :amount="data.amount"></fiat-value>
+              <fiat-value :amount="data.amount" />
```
Comment (venimus) [inline src/components/layout/RightSection.vue]: is this required by the linter?
it's kind of contradicts
https://vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended

Comment (CedrikNikita) [reply src/components/layout/RightSection.vue]: It is not, in this link it is explained very clearly. We don't have a DOM template that is written in html file. We have a single-file components that recommended to close like that <MyComponent/> if there is no content.

**Comment by mradkov** [comment]
Whats the status of this PR? @CedrikNikita

**Comment by mradkov** [comment]
Can you please resolve the conflicts?

**Pull request merged**